opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/Message.java

```diff
@@ -46,14 +50,20 @@
 
 public class Message implements Comparable<Message> {
 
+    @JsonProperty("tags")
     @NotEmpty(message = "tags cannot be empty")
     private Set<String> tags = Collections.singleton(MESSAGES_MAIN_PAGE_TAG);
 
+    @JsonProperty("cssclass")
+    @JsonSerialize(using = HTMLSerializer.class)
```

Comment:
why css class would need encoding? I'm not even sure if that is a legal class name with html encoded entities